software/myfin.yml

```diff
@@ -0,0 +1,37 @@
+---
+name: Addition
+about: Add new software to the list.
+title: Add MyFin
+labels: addition, reviewers wanted
+assignees: ''
+
+---
+
+Thanks for taking the time to suggest an addition to awesome-selfhosted! 
+
+Please fill out information below (all fields are mandatory unless noted otherwise):
+
+```yaml
+name: "MyFin"
+website_url: "https://github.com/afaneca/myfin"
+source_code_url: "https://github.com/afaneca/myfin"
+description: "Personal finances platform (web + rest api + android app) that'll help you budget, keep track of your income/spending and forecast your financial future."
+licenses:
+  - AGPL-3.0
+platforms:
+  - Nodejs
+  - Javascript
+tags:
+  - Money, Budgeting & Management
+depends_3rdparty: false
+demo_url: "https://myfin.afaneca.com"
```

Comment:
This requires credentials which don't seem to be in the README. Is that supposed to be public information?
If not, this should be removed:
```suggestion
```

Reply:
Demo login is 

demo
demo

@afaneca as stated above these should be added to the README so that people can use the demo, When this is done, it can be left in

Reply:
Thanks for pointing that out. Just added it to the readme [here](https://github.com/afaneca/myfin?tab=readme-ov-file#demo-account---try-it-for-yourself).